src/__internal__/form-field/form-field.component.tsx

```diff
@@ -41,6 +41,8 @@ interface CommonFormFieldProps extends MarginProps, ValidationProps {
   reverse?: boolean;
   /** Id of the validation icon */
   validationIconId?: string;
+  /** Flag dedicating if latest validation design should be used */
```

Comment:
If we're going to keep this as prop for now can we use @ignore @private here so it's not in the prop table